client/src/components/Layout/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { SidebarSmall, SidebarBig } from './SidebarVariants';
+
+const links = [
+  { name: 'Strona Główna', url: '/home' },
+  { name: 'Przeliczone puszki', url: '/count' },
+  { name: 'Administracja', url: '/admin' },
+  { name: 'Puszki', url: '/box' },
+];
+
+export const Sidebar: React.FC = () => {
+  const userName = 'superadmin';
+  const isLogin = true;
+
+  return (
+    <>{!isLogin ? <SidebarSmall /> : <SidebarBig links={links} userName={userName} />}</>
```

Comment:
`{isLogin  ? <SidebarBig links={links} userName={userName} />  : <SidebarSmall />}`

Reply:
Could you elaborate more?

Reply:
There is no need to write !isLogin when we're using ternary operator:)